.travis.yml

```diff
@@ -11,8 +11,10 @@ addons:
       - ghc
       - zsh
       - ksh
+      - nodejs
```

Comment:
Updated, please help me terminated the earlier builds on Travis CI if possible, thanks.